docker/docker-compose.yaml

```diff
@@ -76,6 +76,7 @@ services:
       SQLITE_FILE: /app/data/db.sqlite3
       WREN_ENGINE_ENDPOINT: http://wren-engine:${WREN_ENGINE_PORT}
       WREN_AI_ENDPOINT: http://wren-ai-service:${WREN_AI_SERVICE_PORT}
+      OPENAI_GENERATION_MODEL: ${OPENAI_GENERATION_MODEL}
```

Comment:
Can you remove the following line in the docker.go file?
```
// replace OPENAI_API_KEY=sk-xxxxxx with OPENAI_API_KEY=OpenaiApiKey
// replace OPENAI_GENERATION_MODEL= with OPENAI_GENERATION_MODEL=OpenaiGenerationModel
	
```

Reply:
removed comments